resources/airbnb.swiftformat

```diff
@@ -11,7 +11,10 @@
 --closingparen same-line # wrapArguments
 --funcattributes prev-line # wrapAttributes
 --typeattributes prev-line # wrapAttributes
+--beforemarks typealias,class,struct,enum,protocol # organizeDeclarations
+--structthreshold 20 # organizeDeclarations
+--enumthreshold 20 # organizeDeclarations
```

Comment:
@calda do we need a `--classthreshold` too?

Reply:
> If the type in question is a simple value type (e.g. fewer than 20 lines), it is OK to omit the `// MARK:`s, as it would hurt legibility.

My interpretation of the "simple value type" phrasing suggests this exception applies to `struct` and `enum` declarations (value types) and not `class` declarations (reference types). 

Happy to add `--classthreshold 20` as well if you think that is a more correct interpretation of the old rule (perhaps we could reword it as well?).

Reply:
Ah I misread the old rule. I agree that this is the correct interpretation. Thanks for being so thorough!